docs/runbooks/KubeVirtVMStuckInMigratingState.md

```diff
@@ -75,15 +77,13 @@ $ kubectl get nodes -l node-role.kubernetes.io/worker= -o json | jq '.items | .[
 
 ## Mitigation
 
-Ensure you applied the appropriate migration configuration to the VirtualMachine
-according to the nature of the workload.
+Check the migration configuration of the virtual machine to ensure that it is appropriate for the workload. 
+
+You set a cluster-wide migration configuration by editing the `MigrationConfiguration` stanza of the `KubeVirt` custom resource.
 
-Migration configurations can either be set globally via Kubevirt CR's
-`MigrationConfiguration` struct or to a specific scope with
-[Migration Policies](https://kubevirt.io/user-guide/operations/migration_policies/#migration-policies).
-To check whether a VirtualMachine is bound to a migration policy, please refer
-to its `vm.Status.MigrationState.MigrationPolicyName`.
+<!--DS: You set a migration configuration for a specific scope by creating a migration policy.-->
+<!--USstart-->
+You set a migration configuration for a specific scope by creating a [migration policy](https://kubevirt.io/user-guide/operations/migration_policies/#migration-policies).
+<!--USend-->
 
-This problem can be caused by several reasons. Therefore, we advise you to try
-to identify and fix the root cause. If you cannot resolve this issue, please
-open an issue and attach the artifacts gathered in the Diagnosis section.
+You can determine whether a VM is bound to a migration policy by viewing its `vm.Status.MigrationState.MigrationPolicyName` parameter.
```

Comment:
@apinnick Should we also add here the details about how to get help?

Reply:
Sounds good. Will do.